tests/cli.rs

```diff
@@ -0,0 +1,225 @@
+#![cfg(any(target_os = "windows", target_os = "macos", target_os = "linux"))]
+use std::env;
+use std::fs::File;
+use std::io::{BufRead, BufReader};
+use std::path::{Path, PathBuf};
+use std::process::{Child, ChildStderr, Command, Stdio};
+
+use anyhow::{Context, Result};
+use tempfile::tempdir;
+
+#[test]
+fn cli_provide_one_file() -> Result<()> {
+    // provide a path to a file, do not pipe from stdin, do not pipe to stdout
+    test_provide_get_loop(&PathBuf::from("transfer").join("foo.bin"), false, false)
+}
```

Comment:
one other potential change: we can use random data saved to temp files rather than keeping these fixtures around.